artifacts/cni.cncf.io_persistentipamallocations.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.13.0
+  name: persistentipamallocations.cni.cncf.io
+spec:
+  group: cni.cncf.io
+  names:
+    kind: PersistentIPAMAllocation
```

Comment:
Should this be PersistentIPAllocation? Since you are allocating and IP and not an IP management?

I would also consider IPAMLease, since DHCP lease is already a well established term for something very similar, so that can help make the connection for network people. And using IPAM as the prefix can help Kubernetes people connect it with CNIs.

Just 2c, I will not fight against any of the suggested names.

Reply:
What would the short name for ipam lease be ?

Reply:
sgtm

Reply:
Isn't the fact that is not time related an issue with the name ? This is an explicit allocation that will be foreber until VM is gone.

Reply:
that's not actually a bad point. 

According to the dictionary, a lease is time based: 
> a contract by which one party conveys land, property, services, etc. to another for a specified time, usually in return for a periodic payment.

@phoracek the ball's back in your side of the court.

Reply:
At the end of the day we're just 4 people proposing names. The real deal will require the k8snetworkplumbing community involvement. 

I propose we just choose *something* - really, any something - and move this discussion there.

Reply:
I think it's fine, you can lend something to someone and it's on lease until it's returned. Not sure though, we should ask native speakers.

About the short name, I would argue that we don't need any. This resource is not meant for end users. Occupying a short name is wasteful - they should be IMO reserved for resources where short name saves user's time

Reply:
OK, for the sake of any progress, I'll call this IPAM lease, and remove the short name. 

Anyone against it, voice your concerns now. 

As previously stated, anything in here is temporary, and will need broader community consensus. 
@phoracek / @qinqon / @oshoval

Reply:
Fine by me.